app-arch/lz4/lz4-1.8.0.ebuild

```diff
@@ -0,0 +1,35 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit cmake-multilib
+
+if [[ ${PV} == 9999 ]]; then
+	inherit git-r3
+	EGIT_REPO_URI="https://github.com/lz4/lz4.git"
+	EGIT_BRANCH=dev
+else
+	SRC_URI="https://github.com/lz4/lz4/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+	KEYWORDS="~alpha ~amd64 ~arm ~arm64 ~hppa ~ia64 ~m68k ~mips ~ppc ~ppc64 ~s390 ~sh ~sparc ~x86 ~amd64-linux ~x86-linux"
+fi
+
+DESCRIPTION="Extremely Fast Compression algorithm"
+HOMEPAGE="https://github.com/lz4/lz4"
+
+LICENSE="BSD-2 GPL-2"
+SLOT="0"
```

Comment:
Changing subslot to `0` here is certainly wrong. According to [ABI tracker](https://abi-laboratory.pro/tracker/timeline/lz4/), 1.8.0 is not ABI compatible with 1.7.5, so SONAME should be bumped. However, given that the only change was removing stupid `const` from freeing function, I suppose we can ignore that change.